main.ts

```diff
@@ -776,14 +969,13 @@ const issueCommand = new Command()
   .command("start", "Start working on an issue")
   .arguments("[issueId:string]")
   .action(async (_, issueId) => {
+    const teamId = await getTeamId();
+    if (!teamId) {
+      console.error("Could not determine team ID");
+      Deno.exit(1);
+    }
```

Comment:
i think moving this check here would prevent me from doing `linear issue view ABC-123` in random directories, can this be kept how it was?